assemblies/driver/src/assembly/assembly.xml

```diff
@@ -30,6 +30,10 @@
         <include>README.txt</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>src/resources</directory>
+      <outputDirectory>.</outputDirectory>
+    </fileSet>
```

Comment:
Which resources is this bringing in, and what does a user need to do for configuration?

Reply:
Oh sorry, I see the new resource below.  That will need to be on the classpath, right?  Could/should we place the encoder-plugin.xml in the same directory in the zip as the driver jars?

Reply:
I don't see any reason to put it another folder.  It just makes the instructions of how to install more complicated.